Resources/Locale/en-US/wires/wire-names.ftl

```diff
@@ -41,6 +41,7 @@ wires-board-name-flatpacker = Flatpacker
 wires-board-name-spaceheater = Space Heater
 wires-board-name-jukebox = Jukebox
 wires-board-name-computer = Computer
+wires-board-name-barsign = BarSign
```

Comment:
```suggestion
wires-board-name-barsign = Bar Sign
```
No idea why some of these are in PascalCase, but they probably shouldn't be.

Reply:
Because few words names are seem to be that way in here.
``wires-board-name-minigravitygenerator = MiniGravityGenerator``
``wires-board-name-hydroponicstray = HydroponicsTray``
``wires-board-name-telecomserver = TelecomServer``
``wires-board-name-medicalscanner = MedicalScanner``